### DISCUSSION TITLE
Starting with yocto/gl

### DISCUSSION THREAD

**Discussion by pcarret** [xelatihy/yocto-gl#1297] [Q&A] [answered]
Hello,

I have downloaded the repo in VS2019, use CMake, compiled. 
Everything is very fine until I want to start with a sample and I do not rellay know how to start
I went to https://xelatihy.github.io/yocto-gl/ and did not find what I was looking for
I do not know which project to execute first to play with yocto
I would like to process one sample scene for example to start with it.
I did not see any "How to start" in documentation
I'm a bit lost indeed.
Can you help ?

I saw this code below in yocto project but I do not know what parameter I can pass to command line which is empty in VS2019 project by default so it fails when running F5 

```
void run(const vector<string>& args) {
  // command line parameters
  auto error  = string{};
  auto params = app_params{};
  auto cli    = make_cli("yscene", params, "Process and view scenes.");
  if (!parse_cli(cli, args, error)) print_fatal(error);

  // dispatch commands
  if (params.command == "convert") {
    return run_convert(params.convert);
  } else if (params.command == "info") {
    return run_info(params.info);
  } else if (params.command == "render") {
    return run_render(params.render);
  } else if (params.command == "view") {
    return run_view(params.view);
  } else if (params.command == "glview") {
    return run_glview(params.glview);
  } else {
    print_fatal("yscene; unknown command");
  }
}
```

**Comment by giacomonazzaro** [comment]
Hello,
Try setting `yscene` as executable and provide the following command arguments.
It will render the scene `tests/features1/features1.json` and save it to the file `image.png`.
```
render tests/features1/features1.json --output image.png
```

With these arguments instead, the program will open a window running a real-time progressive path tracer

```
view tests/features1/features1.json
```

For these commands to work, you must set the root of the yocto/gl as working directory, otherwise you must provide the absolute path of the scene to render as argument.

**Comment by pcarret** [comment]
Ciao Giacomo,

Many thanks for this help. I launched yscene with view ../../tests/features1/features1.json than screen is this and I do not find where goes the image output and did not view anything coming on screen

![yscene](https://user-images.githubusercontent.com/7167627/136619160-16b86b48-b989-47bc-bea8-eb05589026a1.PNG)
)

Where can I find general info about command line ?
Best regards,
Philippe

**Comment by giacomonazzaro** [comment, marked as answer]
There is something strange. This looks like the output of `yscene` with `render` as command.
With `yscene view` you wouldn't even see the line `render image`.
